test/adapter.test.ts

```diff
@@ -23,7 +24,7 @@ function testGetPolicy(e: Enforcer, res: string[][]) {
 }
 
 test('TestAdapter', async () => {
-    const a = await SequelizeAdapter.newAdapter('mysql://root:@localhost:3306/');
+    let a = await SequelizeAdapter.newAdapter('mysql://root:@localhost:3306/');
```

Comment:
Please write a new test case called ``TestAdapterWithSequelize`` instead of modifying the current one.